sdk/communication/communication-phone-numbers/review/communication-phone-numbers.api.md

```diff
@@ -133,6 +142,38 @@ export interface SearchAvailablePhoneNumbersRequest extends PhoneNumberSearchReq
     countryCode: string;
 }
 
+// @public
+export class SipRoutingClient {
+    constructor(connectionString: string, options?: SipRoutingClientOptions);
+    constructor(endpoint: string, credential: KeyCredential, options?: SipRoutingClientOptions);
+    constructor(endpoint: string, credential: TokenCredential, options?: SipRoutingClientOptions);
+    deleteTrunk(fqdn: string, options?: OperationOptions): Promise<void>;
+    getRoutes(options?: OperationOptions): Promise<SipTrunkRoute[]>;
+    getTrunk(fqdn: string, options?: OperationOptions): Promise<SipTrunk | null>;
+    getTrunks(options?: OperationOptions): Promise<SipTrunk[]>;
+    setRoutes(routes: SipTrunkRoute[], options?: OperationOptions): Promise<SipTrunkRoute[]>;
+    setTrunk(trunk: SipTrunk, options?: OperationOptions): Promise<SipTrunk>;
+    setTrunks(trunks: SipTrunk[], options?: OperationOptions): Promise<SipTrunk[]>;
```

Comment:
Both solutions are probably applicable here. 
I would keep the contract consistent with other languages of the SDK. 

Also, the logic behind is a bit different. 
In case of setTrunk a single trunk PATCH request can be sent. 
But in the case of setTrunks we want to delete all other trunks which were previously stored as well. The PATCH request has to be more complex.
So I would probably keep the flows separate.